slitlessutils/core/wfss/config/instrumentconfig.py

```diff
@@ -656,7 +631,28 @@ def __init__(self, telescope, instrument, disperser, **kwargs):
             raise TypeError(f"disperser is invalid: {disperser}")
 
         d = data['dispersers'][disperser][blocking]
-        self.disperser = load_disperser(disperser, blocking, **d)
+        # d['name'] = disperser
+        # d['blocking'] = blocking
+
+        # DispKey = namedtuple("DispKey", "name blocking")
+        # if isinstance(disperser, str):
+        #     self.disperser = DispKey(disperser, None)
+        # elif isinstance(disperser, (tuple, list)) and len(disperser) == 2:
+        #     self.disperser = DispKey(*disperser)
+        # else:
+        #     raise TypeError(f'Disperser key ({disperser}) is invalid.')
+        # d = data['dispersers'][self.disperser.name][self.disperser.blocking]
+        # d['dispname'] = self.disperser.name
+        # d['blocking'] = self.disperser.blocking
+
+        # d = data['dispersers'][disperser][blocking]
+        # d['name'] = disperser
+        # d['blocking'] = blocking
+
+        self.disperser = load_disperser(d['extraction'], name=disperser, blocking=blocking)
```

Comment:
Looks like this is causing the test failure, it's hitting a `KeyError` because `d` has no "extraction" key. See https://github.com/spacetelescope/slitlessutils/actions/runs/5763932982/job/15626805831?pr=37 for the traceback.

Reply:
Ok...I dug into this error.  This is an error in the test_drizzle file or something.  It is using v1.0.1 of the reference files, and it should be v1.0.2 --- where that keyword does exist.  I'm guessing that the code that triggers the testing scripts, needs to pull the latest version of the reference files.  I think Hattice and/or Duy wrote that?

So, i think we can merge this branch as this isn't an actual problem with the prism code, but the testing code.

Reply:
@Russell-Ryan I am still investigating how the URL is parsed to the versions. But my initial finding is that when the files are downloaded from the static box [URL,](https://github.com/spacetelescope/slitlessutils/blob/fbefd9f2a625bb26d0b6d04e2cfc739252987784/conftest.py#L15) they are downloaded as v1.1.1

Reply:
I'm not entirely sure yet. After running tests locally, I discovered that it pulls version 1.1.1 and runs against the .slitlessutils/1.1.1/instruments/hst_wfc3ir.yaml file, which lacks the "extraction" key.

Reply:
Yea, that makes sense.  I think this is because I had to change the format of the files, and @sosey set up a new box link where we could use direct names.  In moving the files over to the new setup that @sosey made, I think I somehow reset the numbering of the files.  I am fairly sure that anything that points to a static file will be a problem.

Reply:
I'll try to take a look tomorrow.